app/Http/Controllers/Bible/TextController.php

```diff
@@ -232,17 +232,16 @@ public function search()
             return Str::contains($fs->set_type_code, 'audio');
         })->flatten()->toArray();
 
-        $search_text  = '%' . $query . '%';
-
+        $search_text  = \DB::connection()->getPdo()->quote($query);
```

Comment:
does this change maintain the wildcards on both ends, or provide similar capability? It is slow to have a wildcard to begin the query, but it must be maintained